### PR TITLE
Make TreeSigner aware of any previously published STH issued by the same node

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cpp/log/consistent_store.h [hunk 1]
```diff
@@ -140,6 +140,9 @@ class ConsistentStore {
   virtual util::Status AssignSequenceNumber(const int64_t sequence_number,
                                             EntryHandle<Logged>* entry) = 0;
 
+  virtual util::Status GetClusterNodeState(
```
Comment (pphaneuf) [inline cpp/log/consistent_store.h]: How about ```StatusOr```?

Comment (AlCutter) [inline cpp/log/consistent_store.h]: D'oh, yes - thanks.

**Comment by pphaneuf** [comment]
LGTM, pending Travis.

**Comment by pphaneuf** [comment]
Merged as 8dc13067e3aa11c3ff62ae97f1dd60c64c1ab8a5 (the proper way!). :wink:

**Pull request closed without merging**